assets/scss/events.scss

```diff
@@ -1,4 +1,4 @@
-.events-exhibitions {
+.events {
```

Comment:
why are we duping the exact same scss? i would just make it
`.events, .exhibitions` and delete the new file

Reply:
actually, why not keep the grid as `.happenings` which applies to both events + exhibitions?